cleanlab/internal/multilabel_utils.py

```diff
@@ -20,11 +20,12 @@
 
```

Comment:
don't forget to make sure all new files appear at: https://docs.cleanlab.ai/master/ 
Need to add the new helper files to the appropriate index.rst files as mentioned in Development.md guidelines.

Reply:
Added in commit: https://github.com/cleanlab/cleanlab/pull/509/commits/52bfadc9eaa985bf3e63c38b09fed2bd9960acc6